examples/with-solana/src/createSolanaTransfer.ts

```diff
@@ -0,0 +1,62 @@
+import { PublicKey, SystemProgram, Transaction } from "@solana/web3.js";
+import { TurnkeyApi } from "@turnkey/http";
+import { recentBlockhash } from "./solanaNetwork";
+import base58 from "bs58";
+
+/**
+ * Creates a Solana transfer and signs it with Turnkey.
+ * @param fromAddress
+ * @param toAddress
+ * @param amount amount to send in LAMPORTS (one SOL = 1000000000 LAMPS)
+ * @param TurnkeyOrganizationId
+ * @param TurnkeyPrivateKeyId
+ */
+export async function createAndSignTransfer(
+  fromAddress: string,
+  toAddress: string,
+  amount: number,
+  turnkeyOrganizationId: string,
+  turnkeyPrivateKeyId: string
+): Promise<Buffer> {
+  const fromKey = new PublicKey(fromAddress);
+  const toKey = new PublicKey(toAddress);
+
+  const transferTransaction = new Transaction().add(
+    SystemProgram.transfer({
+      fromPubkey: fromKey,
+      toPubkey: toKey,
+      lamports: amount,
+    })
+  );
+
+  // Get a recent block hash
+  transferTransaction.recentBlockhash = await recentBlockhash();
+  // Set the signer
+  transferTransaction.setSigners(fromKey);
+
+  const messageToSign = transferTransaction.serializeMessage();
+
+  const activity = await TurnkeyApi.postSignRawPayload({
+    body: {
+      type: "ACTIVITY_TYPE_SIGN_RAW_PAYLOAD",
+      organizationId: turnkeyOrganizationId,
+      timestampMs: String(Date.now()),
+      parameters: {
+        privateKeyId: turnkeyPrivateKeyId,
+        payload: messageToSign.toString("hex"),
+        encoding: "PAYLOAD_ENCODING_HEXADECIMAL",
+        hashFunction: "HASH_FUNCTION_NOT_APPLICABLE",
```

Comment:
Should we more prominently call out the following?

```
HASH_FUNCTION_NOT_APPLICABLE: Callers must use this enum value when signing with ed25519 keys.\nThis is because, unlike ECDSA, EdDSA's API does not support signing raw digests (see RFC 8032).
```

Reply:
You mean with a comment on top of line 48? I think that's reasonable!